src/platform/silabs/BLEManagerImpl.h

```diff
@@ -25,6 +25,12 @@
 #pragma once
 #if CHIP_DEVICE_CONFIG_ENABLE_CHIPOBLE
 
+#ifdef RSI_BLE_ENABLE
+#define MIN_CONNECTION_INT 45 // 45 msec
```

Comment:
for constants, consider adding unit of measurement: `MIN_CONNECTION_INTERVAL_MS` ... or even better C++ `static constexpr unsigned /* ? */ kMinBleConnectionTimeIntervalMs = 45;` (or maybe consider namespacing to make it clear these are for BLE.)

Reply:
Changed in latest commit